composites/Plugin/ContentAnalysis/components/AnalysisResult.js

```diff
@@ -26,6 +26,12 @@ const AnalysisResultText = styled.p`
 	flex: 1 1 auto;
 `;
 
+let areButtonsHidden = function( props ) {
```

Comment:
This method is returning `true` when the condition in the if is met. But there is no default.